tests/Routing/RoutingRouteTest.php

```diff
@@ -309,24 +309,6 @@ public function testClassesCanBeInjectedIntoRoutes()
         unset($_SERVER['__test.route_inject']);
     }
 
-    public function testClassesAndVariablesCanBeInjectedIntoRoutes()
```

Comment:
Why was this test removed? Makes me a little nervous that perhaps we are breaking something?